src/data_provider/src/sysInfoMac.cpp

```diff
@@ -23,11 +23,13 @@
 #include "ports/portBSDWrapper.h"
 #include "ports/portImpl.h"
 #include "packages/packageFamilyDataAFactory.h"
-#include "packages/pkgWrapper.h"
 #include "packages/packageMac.h"
+#include "sqlite_wrapper.h"
 
 const std::string MAC_APPS_PATH{"/Applications"};
 const std::string MAC_UTILITIES_PATH{"/Applications/Utilities"};
+const std::string MACPORTS_DB_NAME {"registry.db"};
+const std::string MACPORTS_QUERY {"SELECT state, name, version, date, location, archs FROM ports;"};
```

Comment:
We could add a `WHERE` clause to this query, so we can get only `installed` packages and receive fewer data.

This would speed up the loop where we compare against `VALID_STATE`

Reply:
What I was thinking, I used naturally to test the database from the command line in the PR description to provide evidence, and for some reason forgot about it when typing the query in the implementation. 

Thanks!